.github/workflows/release.yml

```
@@ -71,12 +71,66 @@ jobs:
+          published_packages='${{ steps.changesets.outputs.publishedPackages }}'
+
+          if echo "$published_packages" | grep -q '"name": *"gt"'; then
+            echo "should_release_bin=true" >> $GITHUB_OUTPUT
+          else
+            echo "should_release_bin=false" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Extract gt version
+        id: gt-version
+        if: steps.check-gt.outputs.should_release_bin == 'true'
+        run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2086:info:3:28: Double quote to prevent globbing and word splitting

(shellcheck)